projects/baby_driver/scripts/test_i2c_write.py

```diff
@@ -0,0 +1,174 @@
+"""This module tests methods in i2c_write.py"""
+import unittest
+from unittest.mock import patch
+
+from i2c_write import i2c_write
+from message_defs import BABYDRIVER_DEVICE_ID, BABYDRIVER_CAN_MESSAGE_ID
+
+#pylint: disable=unused-argument
+class TestI2CWrite(unittest.TestCase):
+    """Test i2c_write function"""
+
+    @patch('can_util.send_message')
+    @patch('can_util.next_message')
+    #pylint: disable =no-self-use
+
+    def test_i2c_write_parameters(self, mock_next_messaage, mock_send_message):
+        """Tests parameters passed into i2c_write"""
+
+        # Tests minimum values for port, address, tx_bytes and reg
+        mock_next_messaage.return_value.data = [0,0]
+        i2c_write(0,1,[7, 5, 12],1)
+        i2c_write(1,0,[7, 5, 12],1)
+        i2c_write(1,1,[0],1)
+        i2c_write(1,1,[7, 5, 12],0)
+        i2c_write(1,1,[7, 5, 12])
+
+        # Tests maximum values for port, address, tx_bytes and reg
+        i2c_write(1,1,[7, 5, 12],1)
+        i2c_write(0,255,[7, 5, 12],1)
+        i2c_write(0,1,[255, 255, 255],1)
+        i2c_write(0,1,[7, 5, 12],255)
+
+    @patch('can_util.send_message')
+    @patch('can_util.next_message')
+
+    def test_send_message_min(self, mock_next_message, mock_send_message):
```

Comment:
You never test here or in `test_send_message_min` that the command message is as you expect, you should figure out a way to test that.